x-pack/platform/plugins/shared/fleet/server/tasks/sync_integrations_task.ts

```diff
@@ -22,11 +22,13 @@ import { appContextService, outputService } from '../services';
 import { getInstalledPackageSavedObjects } from '../services/epm/packages/get';
 import { FLEET_SYNCED_INTEGRATIONS_INDEX_NAME } from '../services/setup/fleet_synced_integrations';
 
+import { syncIntegrationsOnRemote } from './sync_integrations_on_remote';
+
 export const TYPE = 'fleet:sync-integrations-task';
-export const VERSION = '1.0.0';
+export const VERSION = '1.0.1';
 const TITLE = 'Fleet Sync Integrations Task';
 const SCOPE = ['fleet'];
-const INTERVAL = '5m';
+const INTERVAL = '1m';
 const TIMEOUT = '1m';
```

Comment:
Should we increase this installing a few integrations could take more than 1 minute